style.css

```diff
@@ -104,7 +104,7 @@
   }
 
   main {
-    background-color: #e5e5e5;
+    background-color: #fff6f6b7;
   }
 
   .section-p1 {
```

Comment:
- [ ] Kindly reduce the space between the "headline" section and the "Works" section it is a little bigger compared to the Figma design.
### Your design :point_down: 
![Screenshot from 2023-05-04 11-27-51](https://user-images.githubusercontent.com/91142432/236192721-dba1f9c7-8858-4454-8e26-850ab4266b64.png)
### What it is supposed to look like :point_down: 
![Screenshot from 2023-05-04 11-37-35](https://user-images.githubusercontent.com/91142432/236193084-f0d66814-ae00-4d2d-a499-130a049ddee1.png)